### ISSUE TITLE
Conflicting deprecation warning vs typing of `onInit` in React library

### ISSUE THREAD

**Issue by FoodProduct** [davidjbradshaw/iframe-resizer#1294]
**Describe the bug**
At runtime, I receive the error message
```
Deprecated Option

The onInit() function is deprecated and has been replaced with onReady(). It will be removed in a future version of iFrame Resizer.
```
but the type `IframeResizerProps` has `onInit` and no `onReady`

**To Reproduce**
1. Using `@iframe-resizer/react@5.1.5`, render an `<IframeResizer>` with an `onInit` in the props.
2. Check dev console for warning logs

**Expected behavior**
I expect either:
`onReady` to be an available option in the props
OR
`onInit` to not throw a deprecation warning

**Screenshots**
<img width="511" alt="Screenshot 2024-07-16 at 10 48 14 AM" src="https://github.com/user-attachments/assets/4dcbedb2-1a22-4d5e-90a5-2f83d27771b6">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome
 - Version: 126.0.6478.127

**Additional context**
This might just be a discrepancy between the React version of the library and the underlying parent library. Thanks for the great library!

**Comment by davidjbradshaw** [comment]
Good spot, and just five minutes after I released v5.2 (Arrgh) :) 

I will update the typings to `onReady`.

**Comment by davidjbradshaw** [comment]
Fixed v5.2.1
